pkg/reconciler/v1alpha1/custombuilder/custombuilder.go

```diff
@@ -88,10 +99,21 @@ func (c *Reconciler) reconcileCustomBuilder(customBuilder *experimentalV1alpha1.
 		return v1alpha1.BuilderRecord{}, err
 	}
 
-	return c.BuilderCreator.CreateBuilder(keychain, customBuilder.Spec.CustomBuilderSpec)
+	store, err := c.StoreLister.Get(customBuilder.Spec.Store.Name)
+	if err != nil {
+		return v1alpha1.BuilderRecord{}, err
+	}
+
+	buildPackageStore := &cnb.BuildpackRetriever{
+		Keychain: nil, // TODO: this should be a keychain created from the store spec not the builder spec
```

Comment:
I think our keychain should at least use the k8schain. Maybe initialized in the BuildpackRetriever?